retailer-oas/paths/timeslots/timeslots.yaml

```diff
@@ -11,13 +11,6 @@ get:
   tags:
     - timeslots
   parameters:
-    - name: pickingLocationId
```

Comment:
todo: please is no pl id is involved update the description

Reply:
pls verify